pkg/remote/deploy_gatekeeper.go

```diff
@@ -168,12 +168,12 @@ func generateIngressGatekeeper(codewind Codewind) extensionsv1.Ingress {
 
 	annotations := map[string]string{
 		"nginx.ingress.kubernetes.io/rewrite-target":     "/",
+		"ingress.bluemix.net/redirect-to-https":          "True",
```

Comment:
1. Does adding this in by default affect installs on non-IKS / non-IBM Cloud platforms in any way?
2. We've got the ability to check if: we're running on OpenShift **and** if we're running on IBM Cloud (See function that checks for IBM File storage class). Can we set these annotations only if we're not on OpenShift **but** we're on IBM Cloud?

Reply:
Does not have any effect where that annotation is not available. I tried the changes on IBM Public Cloud + ROKS,  and elsewhere on Openshift 3.11 and Kube in Docker desktop.